src/app/modules/calendar/calendar.component.html

```diff
@@ -0,0 +1,79 @@
+<legend class="ds-c-label--error">
+  <span *ngIf="!valid" class="ds-c-field__hint ds-u-color--error">
+    {{ errorMessage }}
+  </span>
+</legend>
+<div class="ds-l-form-row">
+  <div class="ds-l-col--auto">
+    <label
+      class="ds-c-label ds-u-font-weight--bold ds-u-margin-top--1"
+      for="date"
+    >
+      {{ calendarModel.dateLabel }}
+    </label>
+    <input
+      [matDatepicker]="datePicker"
+      class="datepicker-input"
+      placeholder="DD/MM/YYYY"
+      name="date"
+      id="{{ calendarModel.id + 'date' }}"
```

Comment:
another incredibly nitpicky thing, can you use Date instead of date, so that the id is in camelcase?